vendor/engines/split_accounts/app/models/split_account.rb

```diff
@@ -0,0 +1,28 @@
+class SplitAccount < Account
+
+  has_many :splits, foreign_key: :parent_split_account_id, inverse_of: :parent_split_account
+
+  validate :valid_percent_total
+  validate :one_split_has_extra_penny
+
+  def valid_percent_total
+    unless percent_total == 100
+      errors.add(:splits, "percent total must equal 100")
```

Comment:
Can you move the string to the locales? I think you can do something like `errors.add(:splits, :percent_total`, and then in the locales, set `active_record.errors.splits.percent_total` (I might not have exactly right) in the `en.yml`.